openeo/UDF.ipynb

```diff
@@ -16,6 +16,43 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "\n",
+       "                <script type=\"application/javascript\" id=\"jupyter_black\">\n",
+       "                (function() {\n",
+       "                    if (window.IPython === undefined) {\n",
+       "                        return\n",
+       "                    }\n",
+       "                    var msg = \"WARNING: it looks like you might have loaded \" +\n",
+       "                        \"jupyter_black in a non-lab notebook with \" +\n",
+       "                        \"`is_lab=True`. Please double check, and if \" +\n",
+       "                        \"loading with `%load_ext` please review the README!\"\n",
+       "                    console.log(msg)\n",
+       "                    alert(msg)\n",
+       "                })()\n",
+       "                </script>\n",
+       "                "
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import jupyter_black\n",
+    "%load_ext jupyter_black"
```

Comment:
I would not include this import and `load_ext` in the final notebooks, 
this is unnecessary burden for the end user (they have to install it and understand what it's about)